token_auth/views/register.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib.auth import get_user_model
+from rest_framework import response, decorators, permissions, status
+from rest_framework_simplejwt.tokens import RefreshToken
+
+# from token_auth.models import User
+from token_auth.serializers.register import RegisterSerializer
+
+User = get_user_model()
+
+@decorators.api_view(["POST"])
+@decorators.permission_classes([permissions.AllowAny])
+def register(request):
```

Comment:
We should follow class based views IMO, views? @abha224 @sidvenu

Reply:
Yes @Monal5031 !! I should have kept this in mind. I am updating the commit.